app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -0,0 +1,19 @@
+############################################################################################
+###################################### CONTROLLER ##########################################
+############################################################################################
+
+ListingController = ($scope, $state, ListingService) ->
+  $scope.listings = ListingService.listings
+  $scope.listing = ListingService.listing
+
+############################################################################################
+######################################## CONFIG ############################################
+############################################################################################
+
+ListingModule = angular.module('ListingModule', []);
+ListingController.$inject = ['$scope', '$state', 'ListingService']
+ListingModule.controller 'ListingController', ListingController
+ListingModule.directive 'dahliaShortListing', ->
```

Comment:
@kaptron I wonder if it would make sense to put the directive its own file: see recommended directory structure from the ng-book
![screen shot 2015-11-04 at 4 14 55 pm](https://cloud.githubusercontent.com/assets/5904673/10956285/3f92867e-830f-11e5-9dc1-e3160524c6dd.png)